CONTRIBUTING.rst

```diff
@@ -277,7 +277,7 @@ In another shell:
    cd mlflow/server/js
    yarn start
 
-The MLflow Tracking UI will show runs logged in ``./mlruns`` at `<http://localhost:3000>`_.
+The MLflow Tracking UI will serve runs logged in ``./mlruns`` at `<http://localhost:5000>`_, and the Javascript Dev Server will run at `<http://localhost:3000>`_
```

Comment:
I think  "show runs" is fine. "serve runs" doesn't seem sensible to me as it's the react app that's being served.

Reply:
I thought the react app is being served at 3000 and the MLflow server is at 5000? So it would be `serve runs logged in mlruns from 5000`?

Reply:
@jimmyxu-db thats correct, but "serve runs logged in mlruns from 5000" is still a bit confusing to me. i think we need to explain what's happening at port 3000 and 5000.

should we just be more explicit with something like:
> The react app will run at localhost:3000 and the MLflow server will run at localhost:5000

Reply:
> By default, the react app will run at localhost:3000 and the MLflow server will run at localhost:5000

Makes sense to me! Btw is there actually a way to change these defaults? If so, we can put it here in this doc too.

Reply:
can we use "Javascript Dev Server" instead of "react app" to stay consistent with the rest of the page? Fine with removing "UI" from "MLflow UI server" to avoid confusion.